Add install target

lichtstrahlreiter opened the pull request:
This target installs sysbacklight for all users with setuid as the README suggests.

Commit:
Add install target

This target installs sysbacklight for all users with setuid as the README suggests.

sur5r commented:
While an install target is indeed a useful addition I would like it to support `PREFIX` and `DESTDIR` to be overridable with `PREFIX` defaulting to `/usr/local`. Feel free to update this PR, otherwise I will just leave it open as a reminder for myself to add this.

Commit:
Add install target

This target installs sysbacklight for all users with setuid as the README suggest, supporting 'PREFIX' and 'DESTDIR' with 'PREFIX = /usr/local' as default.

Commit:
Delete Makefile
